build/media/libraries/redcore/media/redcore/css/component.css

```diff
@@ -7938,3 +7938,12 @@
 .redcore .chart-box {
   padding: 0;
 }
+.redcore .redcore .modal-body {
```

Comment:
are you sure this is how it is used? .redcore .redcore .modal-body
maybe this is not needed...